boards/shields/nrf7002_ek/Kconfig.shield

```diff
@@ -4,5 +4,14 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+DT_COMPAT_NORDIC_NRF7002_QSPI := nordic,nrf7002-qspi
+DT_COMPAT_NORDIC_NRF7002_SPI := nordic,nrf7002-spi
+
 config SHIELD_NRF7002_EK
 	def_bool $(shields_list_contains,nrf7002_ek)
+
+config NRF7002_ON_QSPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF7002_QSPI))
+
+config NRF7002_ON_SPI
+	def_bool $(dt_compat_enabled,$(DT_COMPAT_NORDIC_NRF7002_SPI))
```

Comment:
This file is missing a linefeed at the end. Not sure it makes a difference, though.

Reply:
Added LF